### ISSUE TITLE
Add LightGBM option for decision tree

### ISSUE THREAD

**Issue by pkiage** [pkiage/tool-credit-risk-modelling#2]
Why:
Adding LightGBM would enable a comparison of training speed and model effectiveness in comparison to XGBoost.

References:
- [Gradient Boosting with Scikit-Learn, XGBoost, LightGBM, and CatBoost](https://machinelearningmastery.com/gradient-boosting-with-scikit-learn-xgboost-lightgbm-and-catboost/)
- [LightGBM’s documentation](https://lightgbm.readthedocs.io/en/v3.3.2/)
- [LightGBM Github Repo](https://github.com/microsoft/LightGBM)
